.github/workflows/build_and_deploy.yml

```diff
@@ -137,3 +120,55 @@ jobs:
           SLACK_TITLE: Failure deploying release to test
           SLACK_MESSAGE: Failure deploying release to test - Docker tag ${{ needs.build.outputs.docker-image-tag }}
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+  deploy_production:
+    name: Deploy to production environment
+    concurrency: deploy_production
+    runs-on: ubuntu-latest
+    # if: github.ref == 'refs/heads/master' && github.event_name == 'push'
+    needs: [deploy_test]
```

Comment:
I think this needs to be needs: [build,deploy_test] otherwise it can't get the docker_image i.e. needs.build.outputs.docker-image-tag, and will keep installing master.
I think I was the one who said to replace build with deploy_test before! It was almost right...